setup_profile.py

```diff
@@ -0,0 +1,100 @@
+#! /usr/bin/env python3
+
+import os
+import shutil
+from pathlib import Path
+from typing import Union, Tuple
+
+import click
+from rdflib import Graph, FOAF, BNode, Literal
+
+from nanopub import NanopubClient, Nanopub
+from nanopub.definitions import USER_CONFIG_DIR
+from nanopub.namespaces import NPX, ORCID
+from nanopub.java_wrapper import JavaWrapper
+
+PRIVATE_KEY_FILE = 'id_rsa'
+PUBLIC_KEY_FILE = 'id_rsa.pub'
+DEFAULT_PRIVATE_KEY_PATH = USER_CONFIG_DIR / PRIVATE_KEY_FILE
+DEFAULT_PUBLIC_KEY_PATH = USER_CONFIG_DIR / PUBLIC_KEY_FILE
+RSA = 'RSA'
+
+
+@click.command()
+@click.option('--keypair', nargs=2, type=Path,
+              prompt=f'If the public and private key you would like to use are not in {USER_CONFIG_DIR}, '
+                     f'provide them here. If they are in this directory or you wish to generate new keys, '
+                     f'leave empty.',
+              help='Your RSA public and private keys with which your nanopubs will be signed',
+              default=None)
+@click.option('--orcid', type=str, prompt=True, help='Your ORCID')
+@click.option('--name', type=str, prompt=True, help='Your name')
+@click.option('--publish/--no-publish', type=bool, is_flag=True, default=False,
+              help='If true, nanopub will be published to nanopub servers',
+              prompt='Would you like to publish your profile to the nanopub servers?')
```

Comment:
```suggestion
              prompt=('Would you like to publish your profile to the nanopub servers?'
                               'this links your ORCID to your RSA key, thereby making all your'
                               'publications linkable to you'))
```
Or something like that.